src/utils/getTeams.test.ts

```diff
@@ -28,4 +30,28 @@ describe('getTeams', function () {
       'team-enterprise',
     ]);
   });
+
+  it('should return empty array if org is codecov', () => {
```

Comment:
Why does this and the test below have the same behavior? Is this just to verify that codecov is itself an undefined team?

Reply:
Yeah, if any event comes from the codecov org let's just return an empty array for teams temporarily until we roll this out to them